packages/chat-stateful-client/src/EventSubscriber.ts

```diff
@@ -62,7 +62,14 @@ export class EventSubscriber {
   };
 
   private onChatMessageReceived = (event: ChatMessageReceivedEvent): void => {
+    // Today we are avoiding how to render these messages. In the future we can
+    // remove this condition and handle this message appropriately.
+    if (event.type !== 'Text' && event.type !== 'RichText/Html') {
```

Comment:
Is there concerns that we have to do it in stateful client? Since it is our component + selector's responsibility to render the message (and they don't support it well), messages filter should be happening in messageThreadSelector.tsx - In the other hand, stateful client should always provide accurate data directly coming from js sdk (so developers can read messages and render them themselves even without our implementation)

We already have some filterings there in messageThreadSelector.tsx, I wonder why it is not working, I guess there could be some leakings for our conditions there

Reply:
Sync with Alex offline, the big limitation for now is we froze api changes so we are not able to support more types in convertEventType() function above - for short term, this change makes a lot of change before we release the lock of api changes
In long term, we will need to open a bug and add new api changes and move this logic into selector after first GA release